mushroomsoft/app/src/index.html

```diff
@@ -5,9 +5,9 @@
   <title>MushroomSoft IT</title>
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link href="/assets/favicon/favicon.png" rel="shortcut icon" type="image/x-icon">
+  <link href="assets/favicon/favicon.png" rel="shortcut icon" type="image/x-icon">
   <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat:100,100italic,200,200italic,300,300italic,400,400italic,500,500italic,600,600italic,700,700italic,800,800italic,900,900italic" media="all">
-  <link href="/assets/favicon/favicon.png" rel="apple-touch-icon">
+  <link href="assets/favicon/favicon.png" rel="apple-touch-icon">
```

Comment:
use routerLink

Reply:
But it is the index.html in those lines I only call the icon of the tab and the font, when I use routerLink the image and the font are no longer loaded

Reply:
you're right